githubstudent-sanfrancisco.md

```diff
@@ -0,0 +1,6 @@
+# San Francisco, CA
+
+### Things to do
+
+- Golden Gate Bridge
+- GitHub HQ - 88 Colin P Kelley
```

Comment:
@githubstudent I hear GitHub HQ has an Oval Office on the ground floor -- is that true?